pandas/core/generic.py

```diff
@@ -6824,7 +6826,7 @@ def interpolate(
         axis=0,
         limit=None,
         inplace=False,
-        limit_direction="forward",
+        limit_direction=None,
```

Comment:
can you type this (other keywords are a +)

Reply:
not relevant anymore